index.js

```diff
@@ -1314,6 +1314,8 @@ function Runtime() {
         });
 
         return function (target, data) {
+            if (target.isNull()) 
+                throw new Error("The target protocol is null");
```

Comment:
This fixed the following

```
      var NSURLSessionCreation = NSURLSession["+ sessionWithConfiguration:delegate:delegateQueue:"].implementation;
      Interceptor.attach(NSURLSessionCreation, {
        onEnter: function(args) {
          if (!args[3].isNull()) {
            args[3] = new NSURLSessionDelegateProxy(args[3], {});
          }
        }
      });

```

Can't be the parameter being null when there is not delegate ?